scss/bootstrap/variants/_checkbox-switch.scss

```diff
@@ -0,0 +1,114 @@
+.checkbox-switch {
+  padding-top: 0 !important;
+  position: relative;
+
+  input {
+    bottom: 0;
+    height: 0;
+    left: 0;
+    margin: 0;
+    right: 0;
+    top: 0;
+    visibility: hidden;
+    width: 0;
+  }
+
+  :checked {
+    + .checkbox-switch-label {
+      &::after {
+        background-clip: padding-box;
+        margin-left: $crm-checkbox-switch-size-normal;
+      }
+
+      &::before {
+        background-color: $brand-primary;
+        transition: background-color $crm-checkbox-switch-speed ease-in;
+      }
+    }
+  }
+
+  /* Context (colours) */
+
+  &.checkbox-switch-warning {
+    @include checkbox-switch-colour($brand-warning);
+  }
+
+  &.checkbox-switch-danger {
+    @include checkbox-switch-colour($brand-danger);
+  }
+
+  &.checkbox-switch-success {
+    @include checkbox-switch-colour($brand-success);
+  }
+
+  &.checkbox-switch-info {
+    @include checkbox-switch-colour($brand-info);
+  }
+
+  /* Availability */
+
+  [disabled] {
```

Comment:
same here, use `input[disabled]`, disable the linter rule on this line

Reply:
Nested instead